### PR TITLE
fix-datetime64

### PR DESCRIPTION
**Pull request by Prikshit7766**
### Description
[Describe what this change achieves]
 - #338 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -103,16 +103,15 @@ def lint(session):
...
+def test(session):
     session.install(
         "-r",
         "requirements-dev.txt",
         "--timeout",
         "1500",
     )
     session.install(".")
-    session.run("python", "-m", "pip", "install", f"pandas~={pandas_version}")
+    session.run("python", "-m", "pip", "install", "-r", "requirements.txt")
```
Comment (rawwar) [inline noxfile.py:114]: Do we need this?   we are already installing `requirements-dev.txt` in the previous line.

File: noxfile.py [hunk 2]
```diff
@@ -140,12 +140,11 @@ def test(session, pandas_version: str):
...
 
 @nox.session(python=["3.9"])
-@nox.parametrize("pandas_version", ["1.5.0"])
-def docs(session, pandas_version: str):
+def docs(session):
     # Run this so users get an error if they don't have Pandoc installed.
     session.install("-r", "docs/requirements-docs.txt")
     session.install(".")
-    session.run("python", "-m", "pip", "install", f"pandas~={pandas_version}")
+    session.run("python", "-m", "pip", "install", "-r", "requirements.txt")
```
Comment (rawwar) [inline noxfile.py:147]: Do we need this? we are installing requirements on line 145

File: noxfile.py [hunk 3]
```diff
@@ -156,16 +155,15 @@ def docs(session, pandas_version: str):
...
+def trace(session):
     session.install(
         "-r",
         "requirements-dev.txt",
         "--timeout",
         "1500",
     )
     session.install(".")
-    session.run("python", "-m", "pip", "install", f"pandas~={pandas_version}")
+    session.run("python", "-m", "pip", "install", "-r", "requirements.txt")
```
Comment (rawwar) [inline noxfile.py:166]: Do we need this?   we are already installing `requirements-dev.txt` in the previous line.

**Review by rawwar — CHANGES_REQUESTED** [review]
made few comments. please look into them.

**Comment by Prikshit7766** [comment]
@rawwar 
Thanks for reviewing the code. I have updated some of the things that you have suggested. 
let me know if there is anything else.

**Comment by dhrubo-os** [comment]
DCO is missing. Can you please fix the DCO?

**Comment by Prikshit7766** [comment]
@dhrubo-os 
let me know if it works !!

**Comment by rawwar** [comment]
@Prikshit7766 , If you did the sign off correctly, you would see the DCO check passing.  DCO check will mention the commits which are not signed off. So you should be able to verify by yourself next time.

**Comment by Prikshit7766** [comment]
@rawwar 
thanks!! 
I think now DCO check is passing

**Comment by rawwar** [comment]
@Prikshit7766 , Quick question. The issue you linked is only to remove fix the data type. You also included updating the nox tests. Any reason for that?

**Comment by Prikshit7766** [comment]
I have updated ci.yml for installing the requirements.txt. which will install the pandas version mentioned in the requirements.txt.
let me know if it works ?

**Comment by rawwar** [comment]
> I have updated ci.yml for installing the requirements.txt. which will install the pandas version mentioned in the requirements.txt. let me know if it works ?

That's not the fix actually. But, can you just revert those nox related changes and just push the datatype issue? We can take up that nox related one on a separate PR. Also, please run a test locally. You can do it by running `nox -s test` . Make sure to have a opensearch instance running on localhost.

**Comment by Prikshit7766** [comment]
I will close this PR and create a new one

**Pull request closed without merging**